tests/std/tests/P0645R10_text_formatting_formatting/test.cpp

```diff
@@ -1007,6 +1007,239 @@ void test_multibyte_format_strings() {
+    assert(format(STR("{0:>5}"), 42u) == STR("   42"));
+    assert(format(STR("{0:>5}"), -42l) == STR("  -42"));
+    assert(format(STR("{0:>5}"), 42ul) == STR("   42"));
+    assert(format(STR("{0:>5}"), -42ll) == STR("  -42"));
+    assert(format(STR("{0:>5}"), 42ull) == STR("   42"));
+    assert(format(STR("{0:>5}"), -42.0) == STR("  -42"));
+    assert(format(STR("{0:>5}"), -42.0l) == STR("  -42"));
+    assert(format(STR("{0:>5}"), 'c') == STR("    c"));
+    assert(format(STR("{0:>5}"), STR("abc")) == STR("  abc"));
+    assert(format(STR("{0:>8}"), reinterpret_cast<void*>(0xface)) == STR("  0xface"));
+}
+
+template <class charT>
+void libfmt_formatter_test_center_align() {
+    assert(format(STR("{0:^5}"), 42) == STR(" 42  "));
+    assert(format(STR("{0:^5o}"), 042) == STR(" 42  "));
+    assert(format(STR("{0:^5x}"), 0x42) == STR(" 42  "));
+    assert(format(STR("{0:^5}"), -42) == STR(" -42 "));
+    assert(format(STR("{0:^5}"), 42u) == STR(" 42  "));
+    assert(format(STR("{0:^5}"), -42l) == STR(" -42 "));
+    assert(format(STR("{0:^5}"), 42ul) == STR(" 42  "));
+    assert(format(STR("{0:^5}"), -42ll) == STR(" -42 "));
+    assert(format(STR("{0:^5}"), 42ull) == STR(" 42  "));
+    assert(format(STR("{0:^5}"), -42.0) == STR(" -42 "));
+    assert(format(STR("{0:^5}"), -42.0l) == STR(" -42 "));
+    assert(format(STR("{0:^5}"), 'c') == STR("  c  "));
+    assert(format(STR("{0:^6}"), STR("abc")) == STR(" abc  "));
+    assert(format(STR("{0:^8}"), reinterpret_cast<void*>(0xface)) == STR(" 0xface "));
+}
+
+template <class charT>
+void libfmt_formatter_test_fill() {
+    throw_helper(STR("{0:{<5}"), 'c');
+    throw_helper(STR("{0:{<5}}"), 'c');
+    assert(format(STR("{0:*>4}"), 42) == STR("**42"));
+    assert(format(STR("{0:*>5}"), -42) == STR("**-42"));
+    assert(format(STR("{0:*>5}"), 42u) == STR("***42"));
+    assert(format(STR("{0:*>5}"), -42l) == STR("**-42"));
+    assert(format(STR("{0:*>5}"), 42ul) == STR("***42"));
+    assert(format(STR("{0:*>5}"), -42ll) == STR("**-42"));
+    assert(format(STR("{0:*>5}"), 42ull) == STR("***42"));
+    assert(format(STR("{0:*>5}"), -42.0) == STR("**-42"));
+    assert(format(STR("{0:*>5}"), -42.0l) == STR("**-42"));
+    assert(format(STR("{0:*<5}"), 'c') == STR("c****"));
+    assert(format(STR("{0:*<5}"), STR("abc")) == STR("abc**"));
+    assert(format(STR("{0:*>8}"), reinterpret_cast<void*>(0xface)) == STR("**0xface"));
+    assert(format(STR("{:}="), STR("foo")) == STR("foo="));
+    assert(format(basic_string_view<charT>(STR("{:\0>4}"), 6), '*') == basic_string<charT>(STR("\0\0\0*"), 4));
+    throw_helper(STR("{:\x80\x80\x80\x80\x80>}"), 0);
+}
+
+template <class charT>
+void libfmt_formatter_test_plus_sign() {
+    assert(format(STR("{0:+}"), 42) == STR("+42"));
+    assert(format(STR("{0:+}"), -42) == STR("-42"));
+    assert(format(STR("{0:+}"), 42) == STR("+42"));
+    // behavior differs from libfmt, but conforms

```

Comment:
Could we make these trailing comments, so it's clearer that it applies only to a single test case and not all/some of the following test cases?